lib/octicons_angular/src/octicon-component-base.ts

```diff
@@ -11,20 +11,23 @@ export class OpOcticonComponentBase {
   @Input() size:SVGSize = 'medium';
   @Input() verticalAlign = 'text-bottom';
   @Input() title = '';
+  @Input() tabIndex?: number;
 
   @HostBinding('attr.role') role = 'img';
   @HostBinding('attr.fill') @Input() fill = 'currentColor';
   @HostBinding('attr.id') @Input() id = '';
-  @HostBinding('attr.tabindex') @Input() tabIndex:number = -1;
   @HostBinding('attr.aria-label') @Input('aria-label') ariaLabel = '';
   @HostBinding('attr.aria-labelledby') @Input('aria-labelledby') arialabelledby = '';
 
   @HostBinding('class.octicon') baseClassName = true;
   @HostBinding('attr.aria-hidden') get ariaHidden() {
     return !this.ariaLabel;
   }
+  @HostBinding('attr.tabindex') get tabIndexAttr() {
+    return this.tabIndex != null ? this.tabIndex : null;
```

Comment:
Shouldn't this be enough?

```suggestion
    return this.tabIndex;
```